File: AbstractFactory/Java/Product Factory/Creator1.java
public class Creator1 extends AbstractCreator {
    //只生產產品等級為1的A產品
    public AbstractProductA createProductA() {
        return new ProductA1();
    }
    //只生產產品等級為1的B產品
    public AbstractProductB createProductB() {
        return new ProductB1();
    }
}

File: AbstractFactory/PHP/Creator1.php
<?php

include_once ('AbstractCreator.php');
include_once ('ProductA1.php');
include_once ('ProductB1.php');
class Creator1 extends AbstractCreator
{

    public function createProductA(): AbstractProductA
    {
        // TODO: Implement createProductA() method.
        return new ProductA1();
    }

    public function createProductB(): AbstractProductB
    {
        // TODO: Implement createProductB() method.
        return new ProductB1();
    }
}